server/src/server/session/TransactionOLTP.java

```diff
@@ -101,6 +103,8 @@
  */
 public class TransactionOLTP implements Transaction {
     private final static Logger LOG = LoggerFactory.getLogger(TransactionOLTP.class);
+    private long TYPE_SHARD_THRESHOLD;
```

Comment:
I think this should be `final` (as it's initialised in the constructor) and not be all caps as it's not `static`

Reply:
fixed on d67fe2b093939600a77f12ab93fbf6baeb46b5e4